fearless/Modules/Staking/StakingUnbondConfirm/Flow/Parachain/StakingUnbondConfirmParachainViewModelFactory.swift

```diff
@@ -4,47 +4,33 @@ import SoraFoundation
 
 final class StakingUnbondConfirmParachainViewModelFactory: StakingUnbondConfirmViewModelFactoryProtocol {
     let asset: AssetModel
+    let collator: ParachainStakingCandidateInfo
```

Comment:
why viewmodelfactory stores field that should be placed in viewmodelstate ? bondmoreconfirm module also affected